README.md

```diff
@@ -1,68 +1,129 @@
-# sudoers
-> An Ansible role for comprehensive management of a sudoers configuration.
-
-[![Build Status](https://travis-ci.org/wtcross/ansible-sudoers.svg?branch=master)](https://travis-ci.org/wtcross/ansible-sudoers)
+# ahuffman.sudoers
```

Comment:
I'd like to give you contributor access to this repo. That said, I think this'll still need to be changed to wtcross.sudoers.